systemtest/src/test/java/io/strimzi/systemtest/AbstractClusterIT.java

```diff
@@ -253,4 +253,32 @@ void assertNoCcErrorsLogged() {
         //TODO add blacklist for unexpected errors
         assertThat(kubeClient.searchInLog("deploy", "strimzi-cluster-operator", "60", "Exception", "Error", "Throwable"), isEmptyString());
     }
+
+    public List<String> listTopicsUsingPodCLI(String clusterName, String podName) {
+        return asList(kubeClient.exec(podName, "/bin/bash", "-c",
+                "bin/kafka-topics.sh --list --zookeeper " + clusterName + "-zookeeper:2181").out().split("\\s+"));
+    }
+
+    public String createTopicUsingPodCLI(String clusterName, String podName, String topic, int replicationFactor, int partitions) {
+        return kubeClient.exec(podName, "/bin/bash", "-c",
+                "bin/kafka-topics.sh --create --zookeeper " + clusterName + "-zookeeper:2181 " +
+                        "--replication-factor " + replicationFactor + " --partitions " + partitions + " " +
+                        " --topic " + topic + "").out();
```

Comment:
Can we remove the empty string "" or is it some magic =)? It also related to the next methods.